src/processor.ts

```diff
@@ -9,10 +9,14 @@ import {
 } from "@subsquid/substrate-processor";
 import { assertNotNull } from "@subsquid/util-internal";
 import { lookupArchive } from "@subsquid/archive-registry";
-import { events } from "./types";
+import { events, calls } from "./types";
 import { IS_SHIBUYA } from "./mapping/protocolState";
 
-const blockRange = { from: parseInt(process.env.BLOCK_RANGE!, 10) };
+if (!process.env.BLOCK_RANGE) {
+  throw new Error("BLOCK_RANGE is not set");
+}
+
+const blockRange = { from: Number.parseInt(process.env.BLOCK_RANGE, 10) };
```

Comment:
Do you need to use `Number.` here? Perhaps you can use either `parseInt()` or `Number()` here

Reply:
Fixed in https://github.com/AstarNetwork/dapps-staking-indexer-v3/pull/28/commits/7cb07b78b162a6ea624fcc86dc6d3a0a630fbb5d